src/foam/dao/FileJournal.js

```diff
@@ -295,9 +300,21 @@ foam.CLASS({
             if ( COMMENT.matcher(line).matches() ) continue;
 
             try {
-              char operation = line.charAt(0);
-              int length = line.trim().length();
-              line = line.trim().substring(2, length - 1);
+              operation = line.charAt(0);
```

Comment:
A cleaner design would be to create a readCommand() method, which just calls reader.readLine() if multiline is disabled but reads multiple lines if it isn't.